src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -4,6 +4,19 @@ metaDescription: "Errors inbox is an error tracking solution designed to give yo
 redirects:
   - /docs/error-tracking
   - /docs/apm/apm-ui-pages/errors-inbox/errors-inbox
+  - /docs/features/jira-integration
+  - /docs/site/ticketing-integrations
+  - /docs/general/how-do-i-monitor-my-app-in-atlassian-jira
+  - /docs/features/ticketing-integrations
+  - /docs/traces/ticketing-integrations
+  - /docs/apm/new-relic-apm/ui-functions/ticketing-integrations
+  - /docs/accounts-partnerships/education/getting-started-new-relic/ticketing-integrations
+  - /docs/data-analysis/user-interface-functions/ticketing-integrations
+  - /docs/data-analysis/user-interface-functions/share-your-data/ticketing-integrations
+  - /docs/data-analysis/user-interface-functions/share-your-data/ticketing-integrations-jira-lighthouse-pivotal-tracker
+  - /docs/using-new-relic/user-interface-functions/share-your-data/ticketing-integrations-jira-lighthouse-pivotal-tracker
+  - /docs/using-new-relic/user-interface-functions/share-your-data/ticketing-integrations-lighthouse-pivotal-tracker
+  - docs/using-new-relic/user-interface-functions/share-your-data/ticketing-integrations
```

Comment:
@austin-schaefer, this redirect doesn't seem to work in the Gatsby preview. It takes me directly to that ticketing doc.

Reply:
Appears to be some kind of cache issue, reported in #help-deven-websites and they're looking into it.